### ISSUE TITLE
"resulting image is corrupted" w/ v0.18.1 from cargo

### ISSUE THREAD

**Issue by YoshiRulz** [shssoichiro/oxipng#92]
The file in question is [this icon](https://github.com/Regalis11/scpcb/blob/master/GFX/BlinkIcon.png) from the game SCP: Containment Breach. `oxipng -o 6 -s BlinkIcon.png` gives the output:
```
Processing: BlinkIcon.png
    30x30 pixels, PNG format
    1x8 bits/pixel, Grayscale
    IDAT size = 967 bytes
    File size = 1909 bytes
Trying: 648 combinations
Found better combination:
    zc = 4  zm = 8  zs = 1  f = 5        865 bytes
    IDAT size = 865 bytes (102 bytes decrease)
    file size = 1807 bytes (102 bytes = 5.34% decrease)
The resulting image is corrupted and will not be outputted.
This is a bug! Please report it at https://github.com/shssoichiro/oxipng/issues
The resulting image is corrupted
```

**Comment by shssoichiro** [comment]
Thanks! I'll look into it and get it fixed ASAP.

**Comment by shssoichiro** [comment]
Fixed in 0.18.2
